pkg/ui/src/views/jobs/index.tsx

```diff
@@ -200,7 +200,7 @@ const jobsTableColumns: ColumnDescriptor<Job>[] = [
       // If a [SQL] job.statement exists, it means that job.description
       // is a human-readable message. Otherwise job.description is a SQL
       // statement.
-      const additionalStyle = (job.statement ? "" : "--sql");
+      const additionalStyle = (job.statement ? "" : " sql");
```

Comment:
I know very little CSS so sorry if it's a stupid question but shouldn't this be `.sql` ? Can you verify the font is still non-monospace for jobs with description?

Reply:
Doh :/ - thank you for asking this! Just pushed up a fix - RFAL, thank you!

> shouldn't this be `.sql` ?

It doesn't have to be `.sql` on the HTML side: the `.` specifies a className (`.className`), vs an id (`#idName`) -- but you were right to note that the font wasn't monospace for SQL statements! :)

> Can you verify the font is still non-monospace for jobs with description?

Double-checked that font styles are as expected for both SQL and human-readable descriptions:

Human-readable (normal font):
<img width="300" src="https://user-images.githubusercontent.com/3051672/54456101-b8275380-4733-11e9-91f0-75ada828080d.png">

SQL (monospace font):

<img width="300" src="https://user-images.githubusercontent.com/3051672/54456132-cc6b5080-4733-11e9-8869-9c0edb3cc4f7.png">